### PR TITLE
UGENE-6689 Row drag and drop action should be treated as a single action in the Alignment Editor

### PR DESCRIPTION
**Pull request by rasputinkirill**

### CODE REVIEW

File: src/corelibs/U2View/src/ov_msa/MaEditorNameList.cpp [hunk 1]
```diff
@@ -352,6 +354,8 @@ void MaEditorNameList::mousePressEvent(QMouseEvent *e) {
         return;
     }
 
+    U2OpStatus2Log os;
+    changeTracker->startTracking(os);
```
Comment (DmitriiSukhomlinov) [inline src/corelibs/U2View/src/ov_msa/MaEditorNameList.cpp]: But where's **finishTracking()**?

File: src/corelibs/U2View/src/ov_msa/MaEditorNameList.cpp [hunk 2]
```diff
@@ -84,6 +85,7 @@ MaEditorNameList::MaEditorNameList(MaEditorWgt* _ui, QScrollBar* _nhBar)
         connect(editor->getMaObject(), SIGNAL(si_alignmentChanged(const MultipleAlignment&, const MaModificationInfo&)),
             SLOT(sl_alignmentChanged(const MultipleAlignment&, const MaModificationInfo&)));
         connect(editor->getMaObject(), SIGNAL(si_lockedStateChanged()), SLOT(sl_lockedStateChanged()));
+        changeTracker = new MsaEditorUserModStepController(editor->getMaObject()->getEntityRef());
```
Comment (DmitriiSukhomlinov) [inline src/corelibs/U2View/src/ov_msa/MaEditorNameList.cpp]: You need to free the memory in the destructor

File: src/corelibs/U2View/src/ov_msa/MaEditorNameList.cpp (hunk 1 above)
Comment (DmitriiSukhomlinov) [inline src/corelibs/U2View/src/ov_msa/MaEditorNameList.cpp:359]: Where's **finish Tracking**?

File: src/corelibs/U2View/src/ov_msa/MaEditorNameList.cpp (hunk 2 above)
Comment (DmitriiSukhomlinov) [inline src/corelibs/U2View/src/ov_msa/MaEditorNameList.cpp:88]: You need to free the memory in the destructor

**Review by DmitriiSukhomlinov — CHANGES_REQUESTED** [review]

**Review by atiunov — APPROVED** [review]

**Pull request merged**